https://redhat.atlassian.net/browse/ACM-31308 separating more files part 3 (#8884)
open-cluster-management/rhacm-docs: observability/obs_enable_ocp.adoc

> = Enabling observability from the {ocp} console

Reviewer: @dockerymick this one, too -- line 3

[role="_abstract"]

> Optionally, you can enable observability from the {ocp} console. Create a project that is named `open-cluster-management-observability`.

Reviewer: I think this needs more **work in tech debt later**--not now. If you read the description, then you read the steps, we are telling them to _create a project to enable obs._ There is so much more they have to do to enable. This really should be about _why_ the user wants to enable, what the benefit is. Then you can say create the project but really that should be part of the procedure.

> Optionally, you can enable observability from the {ocp} console and create a project with the `open-cluster-management-observability` name.

Author: ```suggestion
Optionally, you can enable observability from the {ocp} console. Create a project that is named `open-cluster-management-observability`.
```

> 

Reviewer: this meaning now changed. 

Now we tell them to take a name and create the project with it. That can mean a lot of things. The first draft was better because it tells them what to name it, not that they can create something with it. 

You can also split this in to two lines. One to enable, and one to create.
